Swat/SwatDetailsView.php

```diff
@@ -517,46 +451,43 @@ protected function validateField(SwatDetailsViewField $field)
         }
     }
 
-    // }}}
-    // {{{ protected function insertField()
-
     /**
-     * Helper method to insert fields into this details-view
+     * Helper method to insert fields into this details-view.
      *
-     * @param SwatDetailsViewField $field the field to insert.
+     * @param SwatDetailsViewField $field           the field to insert
      * @param SwatDetailsViewField $reference_field optional. An existing field
-     *                                               within this details-view to
-     *                                               which the inserted field
-     *                                               is relatively positioned.
-     *                                               If not specified, the
-     *                                               field is inserted at the
-     *                                               beginning or the end of
-     *                                               this details-view's list of
-     *                                               fields.
-     * @param boolean $after optional. If true and a reference field is
-     *                        specified, the field is inserted immediately
-     *                        before the reference field. If true and no
-     *                        reference field is specified, the field is
-     *                        inserted at the beginning of the field list. If
-     *                        false and a reference field is specified, the
-     *                        field is inserted immediately after the reference
-     *                        field. If false and no reference field is
-     *                        specified, the field is inserted at the end of
-     *                        the field list. Defaults to false.
+     *                                              within this details-view to
+     *                                              which the inserted field
+     *                                              is relatively positioned.
+     *                                              If not specified, the
+     *                                              field is inserted at the
+     *                                              beginning or the end of
+     *                                              this details-view's list of
+     *                                              fields.
+     * @param bool                 $after           optional. If true and a reference field is
```

Comment:
The line wrapping got weird here. It is supposed to wrap at 80 chars but this just changed the indentation and did not change the wrapping

Reply:
For docblocks, the code style is trying to keep everything aligned for ease of readability:
- the `@`-tag (`@param`, `@return`, etc.)
- the type (`bool`, `string`, etc.)
- the variable name
- the description
It doesn't always do a great job (especially when the type or variable name is long).  There is also no longer an enforced wrap at 80 characters.  Monitors are bigger, and breaking a long line into two just to fit it in to 80 characters (I find) often makes things less readable than expanding out, to 120 characters or whatever.  In this case, `php-cs-fixer` did the alignment, but doesn't try and re-wrap the comments.